tests/rt.sh

```diff
@@ -393,8 +378,6 @@ elif [[ $MACHINE_ID = hercules ]]; then
   ROCOTOSTAT=$(which rocotostat)
   ROCOTOCOMPLETE=$(which rocotocomplete)
 
-  module use /work/noaa/epic/role-epic/spack-stack/hercules/modulefiles
-  module load ecflow/5.8.4
```

Comment:
`./rt.sh -e ...` fails on Hercules with `ecflow_client: command not found` error  without these two lines.

```
+ ECF_HOST=hercules-login-4.hpc.msstate.edu
+ set +e
+ ecflow_client --ping --host=hercules-login-4.hpc.msstate.edu --port=9563
rt_utils.sh: line 656: ecflow_client: command not found
+ not_running=127
+ [[ 127 -eq 1 ]]
+ echo 'ecflow_server is already running on hercules-login-4.hpc.msstate.edu:9563'
ecflow_server is already running on hercules-login-4.hpc.msstate.edu:9563
+ set -e
+ ECFLOW_RUNNING=true
```

I did not test on other machines.

Reply:
I fixed this in my last commits and tested on Hercules with ecflow.

Reply:
This has been fixed